src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -169,14 +176,23 @@ public Server startServer() throws Exception {
     }
 
     private List<Service> getBeforeHealthCheckServices(Injector injector) {
-        List<Service> services = Stream.of(
+
+        final List<Service> services = Stream.of(
                 InfluxdbReporterLifecycle.class,
                 StatsDReporterLifecycle.class)
                 .map(injector::getInstance)
                 .collect(Collectors.toList());
+
         if (storageBackend == StorageBackend.aws && !appProperties.isReadOnly()) {
             services.add(injector.getInstance(S3WriteQueueLifecycle.class));
+            if (spokeProperties.isTtlEnforced()) {
+                services.add(new SpokeTtlEnforcerService(SpokeStore.WRITE,
+                        injector.getInstance(SpokeTtlEnforcer.class)));
+                services.add(new SpokeTtlEnforcerService(SpokeStore.READ,
```

Comment:
We'll want to be able to run the TTL enforcers in r/o mode, or we could run into the same inode issue for read-heavy batch workloads.